To: vim_dev@googlegroups.com
Subject: Patch 7.3.355
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.355
Problem:    GTK warnings when using netrw.vim. (Ivan Krasilnikov)
Solution:   Do not remove the beval event handler twice.
Files:	    src/option.c


*** ../vim-7.3.354/src/option.c	2011-11-30 11:15:40.000000000 +0100
--- src/option.c	2011-11-30 11:11:15.000000000 +0100
***************
*** 7796,7804 ****
  #ifdef FEAT_BEVAL
      else if ((int *)varp == &p_beval)
      {
! 	if (p_beval == TRUE)
  	    gui_mch_enable_beval_area(balloonEval);
! 	else
  	    gui_mch_disable_beval_area(balloonEval);
      }
  #endif
--- 7796,7804 ----
  #ifdef FEAT_BEVAL
      else if ((int *)varp == &p_beval)
      {
! 	if (p_beval && !old_value)
  	    gui_mch_enable_beval_area(balloonEval);
! 	else if (!p_beval && old_value)
  	    gui_mch_disable_beval_area(balloonEval);
      }
  #endif
*** ../vim-7.3.354/src/version.c	2011-11-30 11:15:40.000000000 +0100
--- src/version.c	2011-11-30 11:30:55.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     355,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
209. Your house stinks because you haven't cleaned it in a week.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
